packages/dashboard/src/features/payments/components/PaymentsSettingsDialog.tsx

```
@@ -550,14 +792,28 @@ export function PaymentsSettingsDialog({ open, onOpenChange }: PaymentsSettingsD
-    configureWebhook.isPending;
+    configureWebhook.isPending ||
+    rzpSaveKey.isPending ||
+    rzpRemoveKey.isPending ||
+    rzpSyncPayments.isPending ||
+    rzpConfigureWebhook.isPending;
+
   const canClose = !isBusy;
   const configuredKeys = keys.filter((key) => key.hasKey);
-  const title = activeTab === 'keys' ? 'Stripe Keys' : activeTab === 'sync' ? 'Sync' : 'Webhooks';
+  const configuredRzpKeys = rzpKeys.filter((key) => key.keyType === 'api_key' && key.hasKey);
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)